src/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <main>
+        <h1>Leader Board</h1>
+        <section class="main-section">
+            <div class="score-list">
+                <div class="score-list-header">
+                 <h2>Recent Scores</h2>
+                <button>Refresh</button>
+                </div>
+             <div class="list-of-scores">
+                <ul>
+                    <li>Nati 220</li>
+                    <li>Nati 220</li>
+                    <li>Nati 220</li>
+                    <li>Nati 220</li>
+                </ul>
+             </div>   
+            </div>
+            <div class="add-score">
+                <h2>Add your Score</h2>
+                <input type="text" name="" id="">
+                <input type="text" name="" id="">
+                <div class="sub-btn">
+                    <button>Submit</button>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] Please make sure that your design matches the following [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), as requested in this milestone.
- [HINT] You can use percentages widths or `vw` units instead of fixed values, which would make your design a little more responsive.
![image](https://user-images.githubusercontent.com/62290545/173847551-7ceb816e-a1c8-4418-abcc-30e6bd651872.png)